content/en/docs/tasks/debug-application-cluster/resource-metrics-pipeline.md

```diff
@@ -1,76 +1,199 @@
+
+It includes all anonymous (non-file-backed) memory since Kubernetes does not support memory swap. The metric also includes some cached (file-backed) memory, because the host OS cannot always reclaim pages.
+
+To learn more about memory, see the [Meaning of memory](https://kubernetes.io/docs/concepts/configuration/manage-compute-resources-container/#meaning-of-memory).
 
 ## Metrics Server
 
-[Metrics Server](https://github.com/kubernetes-sigs/metrics-server) is a cluster-wide aggregator of resource usage data.
-By default, it is deployed in clusters created by `kube-up.sh` script
-as a Deployment object. If you use a different Kubernetes setup mechanism, you can deploy it using the provided
-[deployment components.yaml](https://github.com/kubernetes-sigs/metrics-server/releases) file.
+The metrics-server fetches resource metrics from the kubelets and exposes them in the Kubernetes API server through the Metrics API for use by the HPA and VPA. You can also view these metrics using the `kubectl top` command.
+
+The metrics-server watches for all nodes and pods in your cluster. Nodes for metrics collection discovery; pods for metadata. It uses pod metadata to check pod readiness, and to handle queries with label selectors. 
 
-Metrics Server collects metrics from the Summary API, exposed by
-[Kubelet](/docs/reference/command-line-tools-reference/kubelet/) on each node, and is registered with the main API server via
-[Kubernetes aggregator](/docs/concepts/extend-kubernetes/api-extension/apiserver-aggregation/).
+For example with an HPA query, the metrics-server needs to identify which pods fulfill the label selectors in the deployment.
 
-Learn more about the metrics server in
-[the design doc](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/instrumentation/metrics-server.md).
+The metrics-server employs the Summary API to fetch metrics exposed by
+[kubelet](/docs/reference/command-line-tools-reference/kubelet/) on each node.
 
-### Summary API Source
-The [Kubelet](/docs/reference/command-line-tools-reference/kubelet/) gathers stats at node, volume, pod and container level, and emits their statistics in
+To learn more about the metrics-server, see the [metrics-server repository](https://github.com/kubernetes-sigs/metrics-server).
+
+You can also check out the following:
+
+* [metrics-server design](https://github.com/kubernetes/community/blob/master/contributors/design-proposals/instrumentation/metrics-server.md)
+* [metrics-server FAQ](https://github.com/kubernetes-sigs/metrics-server/blob/master/FAQ.md)
+* [metrics-server known issues](https://github.com/kubernetes-sigs/metrics-server/blob/master/KNOWN_ISSUES.md)
+* [metrics-server releases](https://github.com/kubernetes-sigs/metrics-server/releases)
+* [Horizontal Pod Autoscaling](/docs/tasks/run-application/horizontal-pod-autoscale/)
+
+### Summary API source
```

Comment:
As this endpoint is deprecated we should consider suggesting documenting Resource Metrics endpoint instead `/metrics/resource`

Reply:
We usually document deprecated features. We can (and should) mark things deprecated to make that clear, ideally with an indication of the release of Kubernetes that deprecated it.